tools/common/Driver.cs

```diff
@@ -51,6 +51,7 @@ static void AddSharedOptions (Application app, Mono.Options.OptionSet options)
 			});
 			options.Add ("coop:", "If the GC should run in cooperative mode.", v => { app.EnableCoopGC = ParseBool (v, "coop"); }, hidden: true);
 			options.Add ("sgen-conc", "Enable the *experimental* concurrent garbage collector.", v => { app.EnableSGenConc = true; });
+			options.Add ("interpreter", "Enable the *experimental* interpreter.", v => { app.UseInterpreter = true; });
```

Comment:
This file is shared between mmp and mtouch. I assume you want this option only in mtouch, in which case it should go here: https://github.com/xamarin/xamarin-macios/blob/f7d907983e38e6906a953f9aca4a41fa31cab436/tools/mtouch/mtouch.cs#L967

Reply:
thanks, fixed!